packages/eslint-plugin/src/rules/type-annotation-spacing.ts

```diff
@@ -177,6 +177,22 @@ export default util.createRule<Options, MessageIds>({
       const { before, after } = getRules(ruleSet, typeAnnotation);
 
       if (type === ':' && previousToken.value === '?') {
+        if (sourceCode.isSpaceBetween!(previousToken, punctuatorTokenStart)) {
```

Comment:
`isSpaceBetween` was only added in ESLint v6.7.0
We currently support as far back as ESLint v5.0.0.

Meaning this is not safe.

You need to fall back to the old `sourceCode.isSpaceBetweenTokens()` API for those cases.

Reply:
I saw that `isSpaceBetween` was already used by `object-curly-spacing` so I assumed it was fine to use.